### ISSUE TITLE
use_cli doesn't work in 1.6.0

### ISSUE THREAD

**Issue by o1da** [Azure/terraform-provider-azapi#293]
```
terraform {
  required_version = ">=1.4.2"
  required_providers {
    azurerm = {
      source  = "hashicorp/azurerm"
      version = "3.54.0"
    }
    azapi = {
      source  = "azure/azapi"
      version = "1.6.0"
    }
  }
}

provider "azurerm" {
  subscription_id            = "xxx"
  tenant_id                  = "xxx"
  partner_id                 = "xxx"
  skip_provider_registration = true
  features {}
}

provider "azapi" {
  subscription_id            = "xxx" # G69 DEV
  tenant_id                  = "xxx"
  partner_id                 = "xxx"
  skip_provider_registration = true
  use_cli = true
}
```
Hi,
I am trying to use the code above to make authentication via az CLI working but it's returning:
```
│ Error: Unsupported argument
│
│   on main.tf line 37, in provider "azapi":
│   37:   use_cli = true
│
│ An argument named "use_msi" is not expected here.
```

That's quite weird as I see it's added today https://github.com/Azure/terraform-provider-azapi/blob/2a47a90844b80c70235c3dbec64c6b6f06a365dd/internal/provider/provider.go#L158

Maybe issue with binary pushed to Terraform registry? I am running on Windows 10

**Comment by ms-henglu** [comment]
Hi @o1da ,

Thank you for taking time to report this issue!

Yes, this field is just added to the code base, but not released in the v1.6.0, this feature will be released in v1.7.0. I'll update the changelog.

**Comment by o1da** [comment]
Hi @ms-henglu ,
Thank you for the clarification :)
